backend/app/models/schedule.py

```diff
@@ -0,0 +1,36 @@
+# The Schedule is stored in the database as a string.
+# However, it is created with a JSON object and returns a JSON object
+import uuid
+
+from pydantic import BaseModel
+from sqlmodel import Field, SQLModel
+
+
+class ScheduleData(BaseModel):
+    startDate: str
+    endDate: str | None
+    daysBetween: int
+    skipWeekends: bool
+    skipHolidays: bool
+    timesOfDay: list[str]
+
+
+# Properties to receive on Schedule creation
+class ScheduleCreate(SQLModel):
+    schedule: ScheduleData
+
+
+# Database model
+class Schedule(SQLModel, table=True):
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
+    schedule: str
```

Comment:
Why store schedule as a string?

Reply:
In case the Product Manager wants to change the rules for schedules -- to introduce multiple queries per day or wants to have multiple surveys.

Reply:
I'm not necessarily against it, just not sure I understand. If there were multiple surveys wouldn't that just be another row on the table whether its a string or each field gets its own column? Seems harder to query against strings like that

Reply:
A lot of the work in this PR was serialization and deserializaion of the sring representation.

There are a bunch of attributes of a schedule -- skip weekends, start date,end date, times of day and so on.   
Based on how many times the design has already changed I want to prepare for change in the future. If the PM wants to add on multiple questions per day and that number might be flexible, that's going to be a hard to represent that kind of flexible information in a rigid set of database columns.

I could have had each attribute as its own column, and made the survey id a separate column but I thought this would be more flexible in case the design of survey frequency changed